client/post-editor/editor-sidebar/header.jsx

```diff
@@ -1,11 +1,13 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
 import classnames from 'classnames';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import get from 'lodash/get';
+import noop from 'lodash/noop';
 import { translate } from 'lib/mixins/i18n';
```

Comment:
Wah, this should've never been external (my fault). In any case, two things to note: _If_ we did define this component using `React.createClass`, we could use the global `this.translate` mixin instead. However, more recently I'm evangelizing the [`localize` higher-order component](https://github.com/Automattic/wp-calypso/tree/master/client/lib/mixins/i18n/localize), even for `React.createClass`, as it's much less magical and easier to test, while still being straight-forward to use.